lms/djangoapps/certificates/api.py

```diff
@@ -125,7 +125,7 @@ def get_certificates_for_user(username):
     return certs
 
 
-def get_certificate_for_user(username, course_key):
+def get_certificate_for_user(username, course_key, format_results=True):
```

Comment:
There was already a way to retrieve certificate information from the Certificates app API. It was more useful, at least in the Instructor app, to have access to the certificate itself.